src/test/java/api/loans/scenarios/ChangeDueDateTests.java

```diff
@@ -227,7 +232,7 @@ public void manualDueDateChangeNoticeIsSentWhenPolicyDefinesManualDueDateChangeN
   }
 
   @Test
-  public void itemIsStillDeclaredLostWhenLoanDueDateIsChanged() {
+  public void itemIsStillDeclaredLostAfterAttemptToChangeDueDate() {
```

Comment:
Could this be combined with the test below, by checking that the item is still declared lost after the due date change is rejected?

Reply:
Changed

Reply:
Now that the two tests have been combined, I think `due date cannot be changed when item is declared lost` is a better description of the overall intended behaviour.

```suggestion
  public void dueDateCannotBeChangedWhenItemIsDeclaredLost() {
```

Reply:
Fixed